src/test/unit_tests/jest_tests/test_nc_account_cli.test.js

```diff
@@ -22,6 +22,7 @@ const tmp_fs_path = path.join(TMP_PATH, 'test_nc_nsfs_account_cli');
 const timeout = 50000;
 const config = require('../../../../config');
 const config_fs_account_options = { show_secrets: true, decrypt_secret_key: true };
+const empty_string = "''";
```

Comment:
You add this in the file `manage_nsfs_constants.js` and we reuse it there and in the tests.

Reply:
I don't see where it can be re-used in this form

Reply:
ok, so it is just in the test files (account. bucket).
I would rename it with uppercase, for example:
```suggestion
const UNSET_WITH_EMPTY_STRING = "''";
```